node/forest_libp2p/src/behaviour.rs

```diff
@@ -3,6 +3,8 @@
 
 use super::rpc::{RPCEvent, RPCMessage, RPC};
 use crate::config::Libp2pConfig;
+use forest_cid::Cid;
+use libipld_core::cid::Cid as Cid2;
```

Comment:
Opinionated: Change `Cid2` -> `LibCid` or something other than just the addition of 2. Again opinionated so up to you.

Reply:
I like Cid2. It sounds weird, but I think it better illustrates that we shouldn't be using this library anywhere else... Lemme know, I'm not too emotional about this